e2e/pnpm_lockfiles/v60/pnpm-lock.yaml

```diff
@@ -180,6 +183,24 @@ importers:
         specifier: link:../b
         version: link:../b
 
+  ../projects/peer-types:
+    dependencies:
+      '@scoped/a':
+        specifier: workspace:*
+        version: link:../a
+      '@scoped/b':
+        specifier: link:../b
+        version: link:../b
+      '@scoped/c':
+        specifier: file:../c
+        version: file:../projects/c(@scoped/b@projects+b)
```

Comment:
Notice the `file:` and `(..peers..)` here. That causes a bug as soon as `@scope/c` is used with different peers (such as none).

This is fixed in https://github.com/aspect-build/rules_js/pull/2177